pkgs/by-name/sp/spotifyd/package.nix

```diff
@@ -15,7 +15,7 @@
   withPortAudio ? stdenv.isDarwin,
   portaudio,
   withMpris ? stdenv.isLinux,
-  withKeyring ? false,
+  withKeyring ? true,
```

Comment:
This can be `stdenv.isLinux` I believe

Reply:
The keyring and it's [backend](https://github.com/hwchen/keyring-rs) are cross platform -- you can see them building with this in CI for MacOS [here](https://github.com/Spotifyd/spotifyd/blob/8fb0b9a5cce46d2e99e127881a04fb1986e58008/.github/workflows/ci.yml#L61-L62)

This did make me realize `dbus` will only need to be used on Linux, though :)